### PR TITLE
fix: add declarations

### PR DESCRIPTION
**Pull request by whxaxes**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

Add declarations for supporting typescript

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { SpawnOptions } from 'child_process';
+import { Writable } from 'stream';
+
+declare namespace RunScript {
+  export interface Option extends SpawnOptions {
```
Comment (atian25) [inline index.d.ts]:
```suggestion
  export interface Options extends SpawnOptions {
```

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    stdout: Buffer | null;
+    stderr: Buffer | null;
+  }
+
+  export interface StdError extends Error {
+    stdio: Stdio;
+  }
+}
+
+declare function RunScript(cmd: string, opt?: RunScript.Option): Promise<RunScript.Stdio>;
```
Comment (atian25) [inline index.d.ts]:
```suggestion
declare function RunScript(cmd: string, opts?: RunScript.Options): Promise<RunScript.Stdio>;
```

**Comment by whxaxes** [comment]
这个没其他问题就合一下？

**Comment by atian25** [comment]
@fengmk2  @dead-horse  我没权限发布，git 那些已经提到了，就缺了 publish， 帮忙发下

再顺便给我个权限

```bash
npm owner add atian25 runscript
```

**Comment by atian25** [comment]
1.3.1

**Comment by fengmk2** [comment]
declaration files engineer.

**Pull request merged**